src/_guides/language/effective-dart/usage_migrated.md

```diff
@@ -711,6 +711,24 @@ void error([String? message = null]) {
 
 The following best practices describe how to best use variables in Dart.
 
+### DO follow a consistent rule for `var` and `final` on local variables.
+
+Most local variables shouldn't have type annotations and should be declared
+using just `var` or `final`. There are two rules in wide use for when to use one
+or the other:
+
+*   Use `final` for local variables that are not reassigned and `var` for those
+    that are.
+
+*   Use `var` for all local variables, even ones that are reassigned. Never use
```

Comment:
Is this meant to say "even those that are *not* reassigned"?

Reply:
Oops. I think you're right. @munificent, want me to patch this to "aren't"?

Reply:
Oops, you're right. That would be great, thanks!